storybook/.storybook/main.ts

```diff
@@ -0,0 +1,66 @@
+import type { StorybookConfig } from "@storybook/react-vite"
+import { resolve } from "path"
+import { readFileSync } from "fs"
+
+// Read and parse the webview-ui tsconfig.json to get path mappings
+const webviewTsConfigPath = resolve(__dirname, "../../webview-ui/tsconfig.json")
+const webviewTsConfig = JSON.parse(readFileSync(webviewTsConfigPath, "utf-8"))
+const webviewPaths = webviewTsConfig.compilerOptions?.paths || {}
+
+// Convert webview-ui tsconfig paths to Vite aliases
+const createAliasesFromTsConfig = () => {
```

Comment:
This is complicated, but will save us from duplicating the tsconfig paths here

Reply:
Not a big fan of complicated dynamic stuff, but we can see how this works out for us.